### PR TITLE
Say what units we use.

### PR DESCRIPTION
**Pull request by CMurtagh-LGTM**
Kind of obvious, but still probably needs to be written down.

[Preview](https://deploy-preview-96--nubook.netlify.app/system/foundations/mathematics#units)

### CODE REVIEW

**Comment by KipHamiltons** [comment]
There is a `Conventions and notations` heading in the glossary (#79). It could be worth duplicating this information there too

**Comment by ElderNoSpace** [comment]
> There is a `Conventions and notations` heading in the glossary (#79). It could be worth duplicating this information there too
Are you going to do it or should I?

**Comment by ysims** [comment]
I don't think we are consistent in the code with time units. A lot is in seconds, but there are also some cases of milliseconds. eg script tuner has a const (`defaultDuration`) which is specified in milliseconds.

**Comment by KipHamiltons** [comment]
I'm happy to approve this, with some caveat about time, as per @ysims' comment

**Comment by KipHamiltons** [comment]
@CMurtagh-LGTM finish this bad boy off

File: src/book/02-system/01-foundations/05-mathematics.mdx [hunk 1]
```diff
@@ -207,6 +207,10 @@ If a velocity is multiplied with **time**, it becomes a **vector**.
 Eigen::Vector3d rA_tAb = vAb * t;
 ```
 
+## Units
+
+[S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used, for everything but time. Storage of values may have a different unit, e.g. [nbs and nbs index files](https://nubook.netlify.app/system/foundations/nuclear#nbs-files).
```
Comment (ysims) [inline src/book/02-system/01-foundations/05-mathematics.mdx:212]:
```suggestion
[S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used for everything but time. Storage of values may have a different unit, e.g. [nbs and nbs index files](https://nubook.netlify.app/system/foundations/nuclear#nbs-files).
```
It's so easy to add in more commas, than needed.

File: src/book/03-guides/04-general/03-glossary.mdx [hunk 2]
```diff
@@ -174,6 +174,8 @@ slug: /guides/general/glossary
 
 [**Transformation**](<https://en.wikipedia.org/wiki/Transformation_(function)>): A function which maps a set to itself (sometimes called an operator on that set). Often, it is used in the context of a spatial manipulation - regularly representated as a translation and a rotation.
 
+**Units**: [S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used. Storage of values may have a different unit, e.g. [nbs and nbs index files](https://nubook.netlify.app/system/foundations/nuclear#nbs-files).
```
Comment (ysims) [inline src/book/03-guides/04-general/03-glossary.mdx:177]:
```suggestion
**Units**: [S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used for everything but time. Storage of values may have a different unit, e.g. [nbs and nbs index files](https://nubook.netlify.app/system/foundations/nuclear#nbs-files).
```

Comment (TrentHouliston) [reply src/book/03-guides/04-general/03-glossary.mdx:177]: Even time we generally store as std::chrono, which are unit aware (which is better than having a specific unit)

Comment (JosephusPaye) [reply src/book/03-guides/04-general/03-glossary.mdx:177]: The URL here needs to be updated to nubook.nubots.net

File: src/book/02-system/01-foundations/05-mathematics.mdx [hunk 3]
```diff
@@ -207,6 +207,10 @@ If a velocity is multiplied with **time**, it becomes a **vector**.
 Eigen::Vector3d rA_tAb = vAb * t;
 ```
 
+## Units
+
+[S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used for everything but time. Storage of values may have a different unit, e.g. [nbs and nbs index files](https://nubook.nubots.net/system/foundations/nuclear#nbs-files).
```
Comment (JosephusPaye) [inline src/book/02-system/01-foundations/05-mathematics.mdx:212]: Thinking about these internal URLs, it's best to not include the domain, so the links still work if the domain is different (e.g. for previews, and for local builds).

```suggestion
[S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used for everything but time. Storage of values may have a different unit, e.g. [nbs and nbs index files](/system/foundations/nuclear#nbs-files).
```

File: src/book/03-guides/04-general/03-glossary.mdx [hunk 4]
```diff
@@ -174,6 +174,8 @@ slug: /guides/general/glossary
 
 [**Transformation**](<https://en.wikipedia.org/wiki/Transformation_(function)>): A function which maps a set to itself (sometimes called an operator on that set). Often, it is used in the context of a spatial manipulation - regularly representated as a translation and a rotation.
 
+**Units**: [S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used for everything but time. Storage of values may have a different unit, e.g. [nbs and nbs index files](https://nubook.nubots.net/system/foundations/nuclear#nbs-files).
```
Comment (JosephusPaye) [inline src/book/03-guides/04-general/03-glossary.mdx:177]:
```suggestion
**Units**: [S.I. units](https://en.wikipedia.org/wiki/International_System_of_Units) are used for everything but time. Storage of values may have a different unit, e.g. [nbs and nbs index files](/system/foundations/nuclear#nbs-files).
```

**Review by JosephusPaye — CHANGES_REQUESTED** [review]
Some link changes

**Review by JosephusPaye — DISMISSED** [review]
LGTM

**Review by TrentHouliston — CHANGES_REQUESTED** [review]
I would also mention that in c++ we use the std::chrono library for time which is unit aware. So rather than using SI units, our data has the units attached to them.

**Review by TrentHouliston — APPROVED** [review]

**Pull request merged**